### PR TITLE
moved contact info to its own page

### PR DESCRIPTION
**Pull request by AndySheu**
Created new page (contact info) and linked towards it

### CODE REVIEW

**Comment by gautam-prab** [comment]
Approved, except sponsors ain't gonna just stop by. Keep that as an email and then I'm fine.

**Comment by AndySheu** [comment]
k. I'll change that

File: student-info.md [hunk 1]
```diff
@@ -9,10 +9,7 @@ image: student-info-background
 
 We meet most black days in the LGHS metal shop after school (not on Fridays) from 2-4pm. Feel free to stop by any time. No experience is necessary to be on the robotics team; we only request that you actively participate and contribute your thoughts and skills. The Robotics team isn’t limited to only students interested in engineering: we are also looking for people with various other skills to assist us (such as photography, fundraising, graphic design, public outreach, etc.).
 
-The metal shop can be found in the Industrial Arts wing, which is across the street from the Small Gym and Math/Language building.
-It is listed as room 704 in the "Practical Arts" wing at the bottom of the [LGHS campus map](http://www.lghs.net/about/campusmap.jsp).
-
-If you would like get in contact, you can [email us](mailto:contact@ironclaw972.org) or you can talk to [Mr. Payne](http://www.lghs.net/apps/pages/index.jsp?uREC_ID=323473&type=u), the metal shop teacher.
+Click [here](/contact-info) for contact information.
```
Comment (Joshua-Anderson) [inline student-info.md:12]: This is a little awkward

**Comment by Joshua-Anderson** [comment]
@ClubCharger Make the email a button! Buttonify all of the things!

**Comment by AndySheu** [comment]
Yes! Will do!

**Comment by Joshua-Anderson** [comment]
@ClubCharger LGTM

**Pull request merged**